src/main/scala/templatecontrol/TemplateControl.scala

```diff
@@ -18,21 +19,21 @@ class TemplateControl(config: TemplateControlConfig, githubClient: GithubClient)
     Seq(new CopyTask(config),new FindReplaceTask(config))
   }
 
-  def run(tempDirectory: File, templates: Seq[String]): Future[Seq[ProjectResult]] = {
+  def run(tempDirectory: File, templates: Seq[String], dryRun: Boolean): Future[Seq[ProjectResult]] = {
```

Comment:
dry-run allows us to test locally and be sure that templates are being changed as expected before sending PRs

Reply:
btw, I need to document it. Will do in a while

Reply:
I think I will change it to `no-push`

Reply:
I prefer `dry-run` over `no-push`.

Reply:
But I'm late to the party.